redex.py

```diff
@@ -34,6 +34,17 @@
 from pyredex.utils import abs_glob, make_temp_dir, remove_temp_dirs
 from pyredex.logger import log
 
+# See http://bugs.python.org/issue14315
+def patch_zip_file():
+    old_decode_extra = zipfile.ZipInfo._decodeExtra
+    def decodeExtra(self):
+        try:
+            old_decode_extra(self)
+        except struct.error:
```

Comment:
Our linter is unhappy about this line. It says `undefined name 'struct'`. Does this work on python2 and python3? I haven't had a chance to run this on my machine yet.

Reply:
Just a missing `import struct`. I fixed it